src/api/admin.ts

```diff
@@ -26,6 +27,14 @@ app.post('/sslCerts', async (req, res) => {
     if (stderr) {
       console.log('stderr', stderr)
     }
+    const domains = getDomains()
```

Comment:
you don't want this to happen on error though. Right now you are saving domain even if there is error.